crux-mir/test/conc_eval/intTest/test1451.rs

```diff
@@ -0,0 +1,88 @@
+// A regression test for #1451. This tests the rotate_{left,right} intrinsics
+// at various, commonly-encountered bit widths to ensure that translation
+// succeeds and behaves as expected.
+
+macro_rules! rol_test {
+    ($name:ident, $ty:ty) => {
+        pub fn $name() {
+            let n: $ty = 3;
+            let expected = 12;
+            let actual = n.rotate_left(2);
+            assert!(expected == actual);
+        }
+    }
+}
+
+macro_rules! ror_test {
+    ($name:ident, $ty:ty) => {
+        pub fn $name() {
+            let n: $ty = 12;
+            let expected = 3;
+            let actual = n.rotate_right(2);
+            assert!(expected == actual);
+        }
+    }
+}
+
+rol_test!(i8_rol_test, i8);
+rol_test!(i16_rol_test, i16);
+rol_test!(i32_rol_test, i32);
+rol_test!(i64_rol_test, i64);
+rol_test!(i128_rol_test, i128);
+rol_test!(isize_rol_test, isize);
+
+rol_test!(i8_ror_test, i8);
+rol_test!(i16_ror_test, i16);
+rol_test!(i32_ror_test, i32);
+rol_test!(i64_ror_test, i64);
+rol_test!(i128_ror_test, i128);
+rol_test!(isize_ror_test, isize);
+
+ror_test!(u8_rol_test, u8);
+ror_test!(u16_rol_test, u16);
+ror_test!(u32_rol_test, u32);
+ror_test!(u64_rol_test, u64);
+ror_test!(u128_rol_test, u128);
+ror_test!(usize_rol_test, usize);
```

Comment:
A bit late, but: shouldn't the top 6 macros be `ror_test!` rather than `rol_test!` (and vice versa for the bottom 6)?

Reply:
Oops, quite right. I'll address this in a follow-up PR.

Reply:
https://github.com/GaloisInc/crucible/pull/1454